### PR TITLE
Update Feed Registry docs to latest post-audit

### PR DESCRIPTION
**Pull request by yosriady**
**This PR is ready for review. 

- [x] Redeploy and update contract addresses
  - [x] New post-audit registry: https://etherscan.io/address/0x47Fb2585D2C56Fe188D0E6ec628a38b74fCeeeDf
  - [x] Update docs in this branch
- [x] Update Solidity, JS, Remix, and Hardhat examples after new registry is deployed, and use latest `@chainlink/contracts` https://github.com/smartcontractkit/chainlink/pull/4683/files/7ed29abee800ec9d28643e0c25fdb7bef4212a72..6d1921126b4ab25b6e594e216d3a74d373c1ff31
  - Remix gist: https://gist.github.com/yosriady/9936ee31bc9658b27a2210701e19bc82
  - Hardhat example: https://github.com/smartcontractkit/feed-registry-example
- [x] Expand API reference https://github.com/smartcontractkit/feed-registry/blob/master/contracts/interfaces/FeedRegistryInterface.sol
  - [x] `isFeedEnabled`  
  - [x] Phase / round helpers

### CODE REVIEW

**Comment by yosriady** [comment]
Netlify Preview: https://deploy-preview-202--dreamy-villani-0e9e5c.netlify.app/docs/feed-registry/

**Comment by yosriady** [comment]
TODOs 

- [ ] Add section on Events (specifically FeedConfirmed)
- [ ] Add section on the new round / phase helpers and using it to index events
- [ ] Link to subgraph once that's available as a working example

**Comment by yosriady** [comment]
@dmitrydao Here is the latest doc updates for the Feed Registry.

File: docs/Using Price Feeds/feed-registry.html [hunk 1]
```diff
@@ -11,28 +11,25 @@
...
 
 ![Feed Registry](/files/feed-registry.png)
 
-The Chainlink Feed Registry is an on-chain mapping of assets to feeds. It enables you to query Chainlink price feeds from a pair of asset and denomination addresses directly, without needing to know the feed contract addresses. They enable smart contracts to retrieve the latest price of an asset in a single call, from a single contract.
+The Chainlink Feed Registry is an on-chain mapping of assets to feeds. It enables you to query Chainlink price feeds from asset addresses directly, without needing to know the feed contract addresses. They enable smart contracts to retrieve the latest price of an asset in a single call, from a single contract.
+
+# Base and Quote
 
-# Assets and Denominations
+The Feed Registry maps feeds using `base` and `quote` address pairs as keys. The Feed Registry fully supports the <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol" target="_blank">`AggregatorV3Interface`</a> API, but with additional `base` and `quote` inputs to support multiple pairs and feeds on a single contract.
```
Comment (dmitrydao) [inline docs/Using Price Feeds/feed-registry.html:22]: Please use markdown style for all urls, we have a script that opens all external links in other pages by default.

File: docs/Using Price Feeds/feed-registry.html [hunk 2]
```diff
@@ -55,7 +52,7 @@
 
 ## Denominations library
 
-A <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/Denominations.sol" target="_blank">`Denominations`</a> Solidity library is available for you to query common denominations:
+A <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/Denominations.sol" target="_blank">`Denominations`</a> Solidity library is available for you to query currency identifiers which lack a canonical Ethereum address:
```
Comment (dmitrydao) [inline docs/Using Price Feeds/feed-registry.html:55]: Convert urls to markdown style

File: docs/Using Price Feeds/feed-registry.html [hunk 3]
```diff
@@ -55,7 +52,7 @@
...
 
 
 ```javascript Solidity 8
```
Comment (dmitrydao) [inline docs/Using Price Feeds/feed-registry.html:58]: Replace javascript with solidity for all code samples. We added solidity code styling a while ago.

File: docs/Using Price Feeds/feed-registry.html [hunk 4]
```diff
@@ -69,13 +66,8 @@
...
-  address public constant CNY = address(156);
-  address public constant AUD = address(36);
-  address public constant CAD = address(124);
-  address public constant CHF = address(756);
-  address public constant ARS = address(32);
+  
+  // ... other fiat currencies
 }
 ```
 ```javascript Solidity 7
```
Comment (dmitrydao) [inline docs/Using Price Feeds/feed-registry.html:73]: Do we need Solidity 7? As far as I know, it's a skippable version and most people use either v0.6 or v0.8.

And do we really need to maintain the same code for multiple solidity versions? After all of the Chainlink contracts are moved to v0.8, we plan to maintain the latest version only for all of our products. I think we have to follow the same approach here? Thoughts?

File: docs/Using Price Feeds/feed-registry.html [hunk 5]
```diff
@@ -273,6 +261,16 @@
...
+
+|Network|Address|
+|---|---|
+|Ethereum Mainnet|[`0x47Fb2585D2C56Fe188D0E6ec628a38b74fCeeeDf`](https://etherscan.io/address/0x47Fb2585D2C56Fe188D0E6ec628a38b74fCeeeDf)|
+|Ethereum Kovan|[`0xaa7f6f7f507457a1ee157fe97f6c7db2beec5cd0`](https://kovan.etherscan.io/address/0xaa7f6f7f507457a1ee157fe97f6c7db2beec5cd0)|
+
+
 # API Reference
 
 API reference for <a href="https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/interfaces/FeedRegistryInterface.sol" target="_blank">`FeedRegistryInterface`</a>.
```
Comment (dmitrydao) [inline docs/Using Price Feeds/feed-registry.html:276]: Convert url to markdown style

File: docs/Using Price Feeds/feed-registry.html [hunk 6]
```diff
@@ -286,21 +284,30 @@
...
+|[getNextRoundId](#getnextroundid)| Returns the next round id of a base / quote pair given a specified round.|
+|[getCurrentPhaseId](#getcurrentphaseid)| Returns the current phase id for a base / quote pair.|
 ___
 
+
 ## decimals
 
 Get the number of decimals present in the response value.
 
 ```javascript Solidity
```
Comment (dmitrydao) [inline docs/Using Price Feeds/feed-registry.html:303]: Replace javascript with solidity code tabs for all API functions on the page

**Comment by dmitrydao** [comment]
After all of the comments are fixed, please rebase the branch, and squash your commits

File: docs/Using Price Feeds/feed-registry.html (hunk 4 above)
Comment (yosriady) [reply docs/Using Price Feeds/feed-registry.html:73]: Sure, we can just keep Solidity 8.

**Comment by yosriady** [comment]
@dmitrydao All the comments should be addressed. Can you have another look?

**Comment by dmitrydao** [comment]
> @dmitrydao All the comments should be addressed. Can you have another look?

LGTM. I think @StephenFluin or one of the code owners can take a last look and approve the PR

**Comment by yosriady** [comment]
@StephenFluin @PatrickAlphaC Can I get your review and approval?

**Review by StephenFluin — CHANGES_REQUESTED** [review]
This is super close!

To be consistent with other docs, can we
* Swap code example to testnet (Kovan)
* Add another method like `getEthUsdPrice()` showing how to use the denominations library to get the most common pair (probaly eth/usd since we use this in our other docs)
* Let's clarify maintenance / upkeep expectations on sample repo that we link to https://github.com/smartcontractkit/feed-registry-example
  * When we change the API a year from now, who will keep this up do date? Who will keep the hardhat version up to date? Who manages this when he’s on vacation? Who responds to pull requests & issues? :)

**Comment by yosriady** [comment]
> - Swap code example to testnet (Kovan)
>
> - Add another method like getEthUsdPrice() showing how to use the denominations library to get the most common pair (probaly eth/usd since we use this in our other docs)

Working on it!

> Let's clarify maintenance / upkeep expectations on sample repo that we link to https://github.com/smartcontractkit/feed-registry-example
> * When we change the API a year from now, who will keep this up t date? Who will keep the hardhat version up to date? Who manages this when he’s on vacation? Who responds to pull requests & issues? :)

@PatrickAlphaC Do you think it makes more sense for the feed registry example to live in https://github.com/smartcontractkit/hardhat-starter-kit so the tooling setup and maintenance can be shared? 

I'll work on a PR to add it in.

**Comment by yosriady** [comment]
The code examples have been updated:
- It now uses Kovan 
- It has a new `getEthUsdPrice()` function to demonstrate the use of the `Denominations` library

**Review by StephenFluin — APPROVED** [review]

**Pull request merged**